pkg/cloudprovider/providers/gce/gce_alpha.go

```diff
@@ -23,7 +23,13 @@ import (
 )
 
 // All known alpha features
-var knownAlphaFeatures = map[string]bool{}
+var knownAlphaFeatures = map[string]bool{
+	GCEDiskAlphaFeatureGate: true,
```

Comment:
Can this be controlled by the user, or is it always enabled?

Reply:
Ultimately it is surfaced as a flag in the GCE config file. This is a just a whitelist of what is allowed/accepted from that flag.